src/bokeh/models/sources.py

```diff
@@ -703,7 +703,7 @@ def patch(self, patches: Patches, setter: Setter | None = None) -> None:
                         raise ValueError(f"Initial patch sub-index may only be integer, got: {ind_0}")
 
                     if ind_0 > col_len or ind_0 < 0:
-                        raise ValueError("Out-of bounds initial sub-index (%d) in patch for column: %s" % (ind, name))
+                        raise ValueError("Out-of bounds initial sub-index (%d) in patch for column: %s" % (ind_0, name))
```

Comment:
As long as we are touching this code, this should be converted to an f-string (and also below)

Reply:
Done in fc17688f6